Lib/ufo2ft/__init__.py

```diff
@@ -798,6 +823,28 @@ def compileVariableCFF2s(designSpaceDoc, **kwargs):
     return vfNameToTTFont
 
 
+def _featuresCompatible(designSpaceDoc: DesignSpaceDocument) -> bool:
+    """Returns whether the features of the individual source UFOs are the same.
+
+    NOTE: Only compares the feature file text inside the source UFO and does not
+    follow imports. This will suffice as long as no external feature file is
+    using variable syntax and all sources are stored n the same parent folder
+    (so the same includes point to the same files).
```

Comment:
why can't external  feature file use variable syntax for features to be compatible?

```suggestion
    follow imports. This will suffice as long as all sources are stored n the same
    parent folder (so the same includes point to the same files).
```

Reply:
@madig wrote that bit, any thoughts? I think it should be fine to use variable syntax in external feature files.

Reply:
I may have mistyped. What I meant is that all is good if files are identical and none are using some ifdef kind of text shenanigans.